benchexec/tools/witnesslint.py

```diff
@@ -0,0 +1,21 @@
+import benchexec.result as result
```

Comment:
A license and copyright declaration is needed. In your case, as you are an employee of SoSy-Lab, our standard header applies, you can just copy it from any of the BenchExec files.

Reply:
Added.